doc/requirements.txt

```
@@ -11,4 +11,4 @@ pygments-github-lexers==0.0.5
 qiskit==0.36.0
 sphinxcontrib-bibtex==2.4.2
 sphinx-automodapi==0.14.1
-xanadu-sphinx-theme==0.1.0
\ No newline at end of file
+git+https://github.com/XanaduAI/xanadu-sphinx-theme.git
```

I think we should release a new version of the XST and then use the correct dependency:
```suggestion
xanadu-sphinx-theme==0.2.0
```

---

done!